distribution/server-x/src/main/content/providers/README.txt

```diff
@@ -1 +0,0 @@
-Copy custom providers into this directory then restart the Keycloak server.
```

Comment:
Instead of deleting this README should we not rather add instructions top copy, then run "kc.sh config"?

Reply:
I also changed to show a message when running the `config` step.